mqtt-client-Micropython/pub-tls.py

```diff
@@ -0,0 +1,41 @@
+# pub-tls.py
+import time
+import ussl
+from umqtt.simple import MQTTClient
+
+server = "broker.emqx.io"
+ClientID = f'raspberry-pub-{time.time_ns()}'
+user = "emqx"
+password = "public"
+topic = b'raspberry/mqtt'
+msg = b'{"msg":"hello"}'
+
+def connect():
+    print('Connected to MQTT Broker "%s"' % server)
+    client = MQTTClient(ClientID, server, 8883, user, password, ssl=True, ssl_params={'server_hostname': server})
```

Comment:
Please try to use the following parameters, which will require the client to verify the server certificate:
```
with open('broker.emqx.io-ca.crt', 'rb') as f:
        cadata = f.read()
    ssl_params = dict()
    ssl_params["cert_reqs"] = ssl.CERT_REQUIRED
    ssl_params["cadata"] = cadata
```